EventsExpress/ClientApp/package.json

```diff
@@ -58,10 +58,11 @@
     "ui": "^0.2.4"
   },
   "devDependencies": {
+    "jshint": "^2.12.0",
     "rimraf": "^2.6.2"
   },
   "scripts": {
-    "start": "rimraf ./build && react-scripts start",
+    "start": "rimraf ./build && react-scripts start && jshint ClientApp/src",
```

Comment:
This will only run jshint when you start locally. We need to do the same when you simply build, so we could see the results during CI/CD.